synapse/config/registration.py

```diff
@@ -330,6 +334,19 @@ def generate_config_section(self, generate_secrets=False, **kwargs):
             #email: https://example.com     # Delegate email sending to example.com
             #msisdn: http://localhost:8090  # Delegate SMS sending to this local process
 
+        # If disabled, don't let users set their own display names/avatars
+        # other than for the very first time (unless they are a server admin).
```

Comment:
At first glance it's not super clear what "(unless they are a server admin)" refers to, maybe you could move it to the first part of the sentence to make it extra obvious what it refers to? e.g.:

```suggestion
        # If disabled, don't let users set their own display names/avatars
        # (unless they are a server admin) other than for the very first time.
```